k8s_helm_charts/zdm/values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for zdm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: zdmproxy
+
+origin_username:
+origin_password:
+target_username:
+target_password:
+
+proxy:
+  count: 3
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 512Mi
+    requests:
+      cpu: 1000m
+      memory: 512Mi
+  image:
+    repository: datastax/zdm-proxy
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+cdm:
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 4096Mi
+    requests:
+      cpu: 1000m
+      memory: 2048Mi
```

Comment:
good default would have request memory equal limit memory, I think

Reply:
ok, I'll change them to 2 CPU and 4GB RAM.

Reply:
Should be addressed in the latest commit.